ehcache-example01-clustered-cache/start-client.sh

```diff
@@ -33,7 +33,7 @@ JAVAC="${JAVA_HOME}/bin/javac"
 
 uname | grep CYGWIN > /dev/null && TC_HOME=$(cygpath -w -p "${TC_HOME}")
 
-TC_CP=.
+TC_CP="$WD/src/"
 # Add the client jars in the classpath
 find "${TC_HOME}/client/" -type f -name "*.jar" > file
```

Comment:
Seems the comments I made before about the use of `file` were missed ...  the original comment was about using something like the following to avoid creation of a file:

```
TC_CP="$WD/src"
while IFS= read -r line; do
  TC_CP="${TC_CP}:${line}"
done < <( find "${TC_HOME}/client" -type f -name '*.jar' )
```

The current version creates or overlays a file named `file` in the current directory.  The construct above uses a facility called "process substitution" that uses a pipe instead of a file.  Been available in Bash for a _long_ time.  If you want to keep the file-based scheme, a temporary file should be used -- see mktemp.